### ISSUE TITLE
DataHasValue not recognized in GCI editor

### ISSUE THREAD

**Issue by tholiebig** [protegeproject/protege#566]
According to the OWL spec it should be possible to write GCI's such as:

`dataproperty value "literal"@en SubClassOf class`

However, the GCI editor does not allow for that (the Class Expression Editor for specifying equivalent class axioms, however, does).
Note that Protege will display loaded axioms of this type correctly.

This applies to Protege 5.0 as well as 5.1

**Comment by matthewhorridge** [comment]
If you surround the left hand side with brackets then this should work i.e.
```
(dataproperty value "literal"@en) SubClassOf class
```

**Comment by tholiebig** [comment]
Yes, that works. However, the expression is normalized such that the brackets disappear. When re-editing the axiom is detected as malformed.
